FogCreek/Glitch-Community
# Plump tern: Team User Info popover

## Links
* Remix link: https://plump-tern.glitch.me
* Manuscript link: https://glitch.manuscript.com/f/cases/3328666/

## GIF/Screenshots:
Add as admin
![Screen Shot 2019-06-12 at 1 18 03 PM](https://user-images.githubusercontent.com/938722/59372549-c3c7ed80-8d15-11e9-8dbc-661862faedee.png)

Remove admin
![Screen Shot 2019-06-12 at 1 17 57 PM](https://user-images.githubusercontent.com/938722/59372548-c3c7ed80-8d15-11e9-9f6b-731e64ee4c05.png)

New admin can remove old admin
![Screen Shot 2019-06-12 at 1 32 33 PM](https://user-images.githubusercontent.com/938722/59372956-9fb8dc00-8d16-11e9-91ff-778f19a4123d.png)

Remove user from projects
![Screen Shot 2019-06-12 at 1 18 13 PM](https://user-images.githubusercontent.com/938722/59372550-c3c7ed80-8d15-11e9-8e80-f9a4d1f435e5.png)

all projects selected
![Screen Shot 2019-06-12 at 1 20 07 PM](https://user-images.githubusercontent.com/938722/59372551-c4608400-8d15-11e9-8115-042e7340202e.png)

## Changes:
* combine team-user-info-pop and team-user-remove-pop into single file
* use popover components
* use REST API for matching projects
* update eslint configs to work with implicit labels by nesting https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly

## How To Test:
* log into https://plump-tern.glitch.me
* create a team
* add some projects to the team
* invite another user to join the team
* after the other user has joined the team, the other user should be able to join the other projects
* you should be able to make the other user an admin
* you should be able to remove a user
* if you remove a user who is also a member of any projects, you should be able to remove them from those projects as well
